### ISSUE TITLE
During the training process, the loss is negative

### ISSUE THREAD

**Issue by alu222** [MediaBrain-SJTU/ECGAD#5]
Hello, I am currently studying your article "Multi scale Cross story Framework for Electrocardiogram Anomaly Detection". I would like to ask if I have encountered a situation where the global and local losses are negative when training on my own data. 

“l_global = torch.mean(torch.exp(-global_var)*global_err) + torch.mean(global_var)”
“l_local = torch.mean(torch.exp(-local_var)*local_err) + torch.mean(local_var)”

Have you encountered this situation during training? If the loss of the first epoch is already negative, will it continue to converge in the future？
